frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -471,8 +471,8 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
                     )}`
                 )
                 const counts = response.result?.[0]?.data
-                const firstWeek = counts.slice(0, 7)
-                const avg = Math.round(sum(firstWeek) / 7)
+                console.log(counts)
+                const avg = Math.round(sum(counts) / 7)
```

Comment:
Is this expected? Can the filter be > 7 days?

Reply:
Yep! If you query back 7 days but it's 5pm you'll actually get 8 days worth of data as it starts at 5pm

Super simple example: if you do 1 day back and it's 5pm you'll get yesterday at 5pm to midnight and today midnight to 5pm which gives two days